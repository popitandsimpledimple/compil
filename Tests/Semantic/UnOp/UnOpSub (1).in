var a: boolean;

begin
    write(-a);
end.