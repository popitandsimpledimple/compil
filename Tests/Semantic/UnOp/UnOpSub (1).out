(4,11) Unary operator is not overloaded: Subboolean